CHANGELOG/v1.8.0-beta.0.md

```diff
@@ -0,0 +1,401 @@
+
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.0
+## :chart_with_upwards_trend: Overview
+- 259 new commits merged
+- 1 breaking change :warning:
+- 16 feature additions ✨
+- 33 bugs fixed 🐛
+
+## :warning: Breaking Changes
+- ClusterClass: Adjust CC & Cluster controller to block on variable conflicts, deprecate definitionFrom (#10841)
+
+## :sparkles: New Features
+- API: Add support negative polarity conditions (#10550)
+- API: Optimize rbac across controllers (#10552)
+- Bootstrap: Allow CAPBK to generate JoinConfiguration discovery kubeconfig (#10799)
+- CABPK: Add ExtraEnvs and ImagePullSerial to KubeadmConfig (#10846)
+- CABPK: Add kubeadm v1beta4 types (#10709)
+- ClusterClass: Add map key for MD/MP class & topology in ClusterClass & Cluster.spec.topology (#10808)
+- ClusterClass: Add metadata for top-level and nested variables & deprecate previous metadata field (#10778)
+- ClusterClass: Add variable validation to ClusterClass controller, block Cluster reconcile if variables not reconciled (#10812)
+- ClusterClass: Extend ControlPlane/ MD / MP builtin to include metadata (#10837)
+- ClusterClass: Introduce CEL for ClusterClass Variables (#9239)
+- ClusterClass: Support ControlPlane variable overrides (#10682)
+- e2e: Test: Bump Kubernetes version used for testing to v1.30.0-rc.1 (#10384)
+- MachineSet: Add remediation strategy support in MachineDeployment (#10712)
+- Runtime SDK/ClusterClass: Extend cluster builtin to include UID (#10822)
+- Runtime SDK/ClusterClass: Fix GetObjectVariableInto util func (#10702)
+- Testing: Support e2e for WSL (#10402)
+
+## :bug: Bug Fixes
+- API: Re-add patch for events (#10695)
+- CAPD: Don't add host ports to exposed ports of containers (#10657)
+- CAPD: Verify lb config after writing it (#10453)
+- CI: Add for new linters to fix old syntax (#10699)
+- CI: Add tenv linter (#10689)
+- CI: Fix conversion-gen outside of GOPATH (#10502)
+- ClusterClass: Defaulting webhook should check class is set in ClusterClass-based clusters (#10671)
+- ClusterClass: Fix indexing in ClusterClass webhook MHC validation (#10672)
+- clusterctl: Ensure cert-manager objects get applied before other provider objects (#10469)
+- clusterctl: Fix log in clusterctl move (#10540)
+- clusterctl: Verify that there is a release for the tag (#10220)
+- ClusterResourceSet: Correctly handle concurrent updates to ClusterResourceSetBinding (#10656)
+- ClusterResourceSet: Crs: use separate cache for partial metadata watches on secrets to include all secrets (#10633)
+- Dependency: Use k8s.io/utils/ptr instead of k8s.io/utils/pointer (#10700)
+- e2e: Fix kubetest to allow parallel execution on different clusters (#10424)
+- e2e: Kubetest: also gather junit reports in case of errors observed from ginkgo (#10493)
```

Comment:
```suggestion
- e2e: Kubetest also gather junit reports in case of errors observed from ginkgo (#10493)
```

Reply:
@vishalanarase @chandankumar4 Let's please keep these kind of colons the next time. It breaks the readability otherwise